src/roswire/common/launch/controller.py

```diff
@@ -10,7 +10,7 @@
 
 @attr.s(frozen=True, slots=True, auto_attribs=True)
 class ROSLaunchController:
```

Comment:
Should we rename to `LaunchController`?

Reply:
I thought that could remain because it is the Launch Controller for all versions of ROS

Reply:
That's fair.